.github/workflows/rust-ci-cd.yml

```
@@ -571,7 +593,20 @@ jobs:
       - test-prowo-edge-cases
       - test-performance
       - test-prowo-docker
-    if: needs.allow-deploy.outputs.RUN_DEPLOY == 'true'
+    if: |
+      always() && 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)